TestFoundation/TestDecimal.swift

```diff
@@ -277,7 +277,9 @@ class TestDecimal: XCTestCase {
                 }
             }
         }
+#if !arch(arm)
         XCTAssertEqual(Decimal(186243*15673), Decimal(186243) * Decimal(15673))
+#endif
```

Comment:
Why is this not possible on Android?

Reply:
If it's a 32-bit vs 64-bit thing, can we express that differently so it works on other 32 bit platforms?

Reply:
Couldn't get it to compile for some reason even though there is Decimal.init(_ value: Int64)

Reply:
Error was:
```
/home/johnno/swifty-robot-environment/util/prepare_environment/test-foundation/Sources/TestDecimal.swift:280:38: error: arithmetic operation '186243 * 15673' (on type 'Int') results in an overflow
        XCTAssertEqual(Decimal(186243*15673), Decimal(186243) * Decimal(15673))
```

Reply:
Was able to fix this: Decimal(Int64(186243)*Int64(15673)). I had a second look and can’t see how the other commenting out could be avoided.

Reply:
This is an incorrect use of these initializers. `Int64(15673)` creates a value of type `Int` from the literal and then converts it to a value of type `Int64`; indeed, consideration has been given to making this usage with literals a compiler warning. You want to write instead `Decimal(186243 * 15673 as Int64)`.

Reply:
Thanks @xwu. Resolved.